code/datums/datum.dm

```diff
@@ -0,0 +1,2 @@
+/datum
+	var/datum/weakref/weak_reference
```

Comment:
Is there a reason this needs to be a `datum` level var?

Reply:
The point is to cache it for future accesses, I suppose?

Reply:
Right.